src/meta/src/hummock/vacuum.rs

```diff
@@ -91,16 +95,17 @@ where
                 self.hummock_manager
                     .delete_will_not_be_used_ssts(*version_id, &ssts_in_use)
                     .await?;
-                // Delete version metadata and mark SST as orphan (set meta_delete_timestamp).
-                // TODO delete in batch
                 let stale_ssts_left = self.hummock_manager.get_ssts_to_delete(*version_id).await?;
                 if stale_ssts_left.is_empty() {
-                    self.hummock_manager.delete_version(*version_id).await?;
-                    vacuum_count += 1;
+                    versions_to_delete.push(*version_id);
```

Comment:
should we do `delete_versions` here when `versions_to_delete` reaches `batch_size` instead of `list_version_ids_asc(Some(batch_size))`